src/groupInfo/components/GroupGalleryUI.vue

```diff
@@ -40,6 +40,13 @@
         class="text-primary header"
       >
         {{ $t('JOINGROUP.WHICHGROUP') }}
+        or
```

Comment:
I guess that should be translatable. Maybe use the i18n functional component to provide more context/give more freedom to translators? e.g. `{joinGroup} or {createGroup}`

Reply:
Oh yeah, you mean not everyone says _or_? So complicated!